src/runners/JasmineComplianceRunner.ts

```diff
@@ -40,24 +41,32 @@ const defaultJasmineConfigurer = (jasmine) => {
   
     // env.fail
     env.clearReporters(); // Clear default console reporter
-    env.specFilter = () => true
+    // Spec filter on env is deprecated
+    // env.specFilter = () => true
     let reporter = new TSConsoleReporter()
     reporter.setOptions( {showColor : false})
     env.addReporter(reporter);
     //  env.throwOnExpectationFailure(true)
-  
 }
 
 export class JasmineComplianceRunner {
     jasmine
     
-    constructor(callback: Callback) {
+    constructor(callback: Callback, productname?:string, s3BucketName?:string, localExecution?:boolean ) {
```

Comment:
Consider using variable args for the reporter to add reporter eg; constructer(...reporters: Reporters[])

now that I can think about it... we can could remove the callback.
Instead of doing it there I'd probably make a function that returns a promise<void> of test completion eg...

eg Execute(): Promise<void> to indicate completion. When constructing, add a 'fake' reporter that listens for the completion, and return a promise that resolves when the fake reporter exits.

then inside the handler you can do `await runner.Execute()` instead of passing the callback in.